### ISSUE TITLE
How We use Common SDK for two Apps ?

### ISSUE THREAD

**Issue by DheemanS** [mean-expert-official/loopback-sdk-builder#504]
#### What type of issue are you creating?
- [ ] Bug
- [ ] Enhancement
- [x] Question

#### What version of this module are you using?
- [ ] 2.0.10 (Stable)
- [x] 2.1.0-rc.n (2.1 Release Candidate n)
- [ ] Other

Write other if any:

#### Please add a description for your issue:

Are we able to use common SDK for two Apps ?

**Comment by Legited** [comment]
Yes, you can import same module to however many apps you want. Import the same SDK module into other apps. As long as you don't break the path to your shared folder you will be ok

**Comment by DheemanS** [comment]
Thanks Man

**Comment by DheemanS** [comment]
Currently I am facing a issue with login 
User is login in App1 but when we run App2 in same browser then it's show us already login in App2.

**Comment by Legited** [comment]
This is not an SDK issue, sounds like you are somehow sharing the same service between Angular applications. What I meant earlier is that you can use the same generated SDK to build as many app as you want

**Comment by Destreyf** [comment]
In order for different app's to have different credentials you'll need to change the local storage prefix, this can be found here: `sdk/services/core/auth.service.ts` set `prefix` to be different for each app, this means you can't share the entire SDK between applications, or you could add your own method to change this in your app bootstrap, it could also be wrapped up into the `LoopBackConfig` as a configuration option.

@jonathan-casarrubias i wonder if we should make this exposed either on the LoopBackAuth service, or in the LoopBackConfig

**Comment by jonathan-casarrubias** [comment]
@Destreyf because of this topic I just found that there already is a configuration method for defining  the prefix within the LoopBackConfig, but it seems it is not connected to the LoopBackAuth, the latter always uses a hardcoded prefix, I think is just matter of connecting the config
